pkgs/tools/backup/partclone/default.nix

```diff
@@ -1,18 +1,19 @@
-{stdenv, fetchurl
+{stdenv, fetchFromGitHub
 , pkgconfig, libuuid
-, e2fsprogs
+, e2fsprogs, automake, autoconf
 }:
 stdenv.mkDerivation {
   name = "partclone-stable";
   enableParallelBuilding = true;
 
-  src = fetchurl {
-    url = https://codeload.github.com/Thomas-Tsai/partclone/legacy.tar.gz/stable;
-    sha256 = "12bnhljc4n4951p5c05gc7z5qwdsjpx867ad1npmgsm8d9w941sn";
-    name = "Thomas-Tsai-partclone-stable-20150722.tar.gz";
+  src = fetchFromGitHub {
+    owner = "Thomas-Tsai";
+    repo = "partclone";
+    rev = "stable";
```

Comment:
rev = "stable"? That doesn't sound like a good ref to me (doesn't it "move"?). How about a tag or commit id?